types/react-mathquill/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for react-mathquill 0.1
+// Project: https://github.com/viktorstrate/react-mathquill#readme
+// Definitions by: Marco Gonzalez <https://github.com/magonzalez9>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.5
+
+import { ComponentProps, ReactNode, Component } from 'react';
+import { MQ, Config } from 'edtr-io__mathquill';
```

Comment:
Please use the original name of the package.

```suggestion
import { MQ, Config } from '@edtr-io/mathquill';
```